### DISCUSSION TITLE
Question: Teleport Support integration with AWS Identity Center (Old AWS SSO)?

### DISCUSSION THREAD

**Discussion by McLaouth** [gravitational/teleport#48436] [General]
I need to know if Teleport, in its free or paid version, is compatible with the AWS Identity Center (the old AWS SSO), since we are currently not using the IAM service because it is less robust than the Identity Center.

Could you please help me confirm this concern?

Thank you very much

**Comment by webvictim** [comment]
Teleport Enterprise (paid version) has support for AWS Identity Center coming very soon: https://goteleport.com/docs/upcoming-releases/#preview-aws-identity-center-integration

**Comment by benarent** [comment]
We added this feature into Teleport 17.
- Overview Video https://www.youtube.com/watch?v=VGPt_As0p9w
- Docs https://goteleport.com/docs/admin-guides/management/guides/aws-iam-identity-center/
